CMakeLists.txt

```diff
@@ -51,10 +51,36 @@ set(LULESH_SOURCES
   lulesh-viz.cc
   lulesh.cc)
 
+set(LULESH_FOR_LOOP_SOURCES
+  lulesh-comm.cc
+  lulesh-init.cc
+  lulesh-util.cc
+  lulesh-viz.cc
```

Comment:
The lulesh-*.cc are commonly used in three executable. To avoid code duplication and improve code re-utilization, can you pack lulesh-*.cc into one cmake macro? Something like:
```cmake
set(LULESH_SOURCES
   lulesh-comm.cc
   lulesh-init.cc
   lulesh-util.cc
   lulesh-viz.cc)

set(LULESH_FOR_LOOP_SOURCES 
   LULESH_SOURCES
   lulesh_for_loop.cc
```

Reply:
Cool, nice idea!. Will do that. Thanks